src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  
+  title = 'AngularApp';
```

Comment:
could you please clarify why we need it?

Reply:
Tried some testing and forgot to remove - done, together with respective test